test/controllers/howDoYouWantToPay.controller.test.ts

```diff
@@ -0,0 +1,122 @@
+import 'reflect-metadata'
+
+import { assert } from 'chai'
+import { Application } from 'express'
+import { StatusCodes } from 'http-status-codes'
+import request from 'supertest'
+import { anything, instance, mock, verify, when } from 'ts-mockito'
+import { generateHowDoYouWantToPayForm } from '../fixtures/payment.fixtures'
+import { generateDissolutionSession } from '../fixtures/session.fixtures'
+import { createApp } from './helpers/application.factory'
+
+import 'app/controllers/howDoYouWantToPay.controller'
+import ApplicationStatus from 'app/models/dto/applicationStatus.enum'
+import DissolutionGetResponse from 'app/models/dto/dissolutionGetResponse'
+import PaymentType from 'app/models/dto/paymentType.enum'
+import DissolutionSession from 'app/models/session/dissolutionSession.model'
+import { HOW_DO_YOU_WANT_TO_PAY_URI, PAY_BY_ACCOUNT_URI, SEARCH_COMPANY_URI } from 'app/paths'
+import DissolutionService from 'app/services/dissolution/dissolution.service'
+import PaymentService from 'app/services/payment/payment.service'
+import SessionService from 'app/services/session/session.service'
+
+import { generateDissolutionGetResponse } from 'test/fixtures/dissolutionApi.fixtures'
+
+describe.only('HowDoYouWantToPayController', () => {
+
+  let dissolutionService: DissolutionService
+  let paymentService: PaymentService
+  let sessionService: SessionService
+
+  const TOKEN = 'some-token'
+  const COMPANY_NUMBER = 'ABC123'
+  const REDIRECT_CARD_URL = 'http://card-payment-ui-url'
+
+  let dissolution: DissolutionGetResponse
+  let dissolutionSession: DissolutionSession
+
+  before(() => {
+    dissolutionService = mock(DissolutionService)
+    paymentService = mock(PaymentService)
+    sessionService = mock(SessionService)
+
+    dissolution = generateDissolutionGetResponse()
+    dissolutionSession = generateDissolutionSession(COMPANY_NUMBER)
+
+    when(sessionService.getAccessToken(anything())).thenReturn(TOKEN)
+    when(sessionService.getDissolutionSession(anything())).thenReturn(dissolutionSession)
+    when(paymentService.generatePaymentURL(TOKEN, dissolutionSession, anything())).thenResolve(REDIRECT_CARD_URL)
+    when(dissolutionService.getDissolution(TOKEN, dissolutionSession)).thenResolve(dissolution)
+  })
+
+  function initApp(): Application {
+    return createApp(container => {
+      container.rebind(DissolutionService).toConstantValue(instance(dissolutionService))
+      container.rebind(SessionService).toConstantValue(instance(sessionService))
+      container.rebind(PaymentService).toConstantValue(instance(paymentService))
+    })
+  }
+
+  describe('GET - ensure that the how do you want to pay page loads correctly', () => {
```

Comment:
The describe should just say `GET` or `POST`. Your first test case contradicts this sentence anyway